MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  QuestionFactory.swift
+//  MovieQuiz
+//
+//  Created by Арсений Убский on 15.11.2022.
+//
+
+import Foundation
+
+final class QuestionFactory: QuestionFactoryProtocol {
+    private let moviesLoader: MoviesLoading
+    weak private var delegate: QuestionFactoryDelegate?
+    init(moviesLoader: MoviesLoading,delegate: QuestionFactoryDelegate?) {
```

Comment:
Так будет аккуратнее
```suggestion
    init(moviesLoader: MoviesLoading, delegate: QuestionFactoryDelegate?) {
```